src/ellipticLCS.jl

```diff
@@ -611,16 +611,16 @@ function compute_returning_orbit(
     maxiters::Int = 2000,
     tolerance::Float64 = 1e-8,
     max_orbit_length::Float64 = 20.0,
-) where {T<:Real}
+)::Tuple{Vector{SVector{2,T}},Int} where {T<:Real}
```

Comment:
This type assertion seems to be necessary for successful type inference... for some reason.